source/Tests/Testing/Integration/IntegrationTestForwardCancellationTokenToInvocationsAnalyzerFixture.cs

```diff
@@ -0,0 +1,426 @@
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Testing;
+using NUnit.Framework;
+using Octopus.RoslynAnalyzers;
+using Verify = Tests.CSharpVerifier<Octopus.RoslynAnalyzers.Testing.Integration.IntegrationTestForwardCancellationTokenToInvocationsAnalyzer>;
+
+namespace Tests.Testing.Integration
+{
+    public class IntegrationTestForwardCancellationTokenToInvocationsAnalyzerFixture
+    {
+        [TestCase]
+        public async Task No_Diagnostic_ClassesThatHaveNothingToDoWithThisAnalyser()
```

Comment:
nit - file name uses the American `Analyzer` spelling (as does the project) but this method using the British `Analyser`.